packages-native/safe-area-view/README.md

```diff
@@ -0,0 +1,51 @@
+# Safe area view widget
+
+Prevents content being rendered in an unsafe areas like rounded screen corners or a notch. Currently, this widget only
+supports iOS.
+
+## Custom styling
+
+You can define your custom styles through Atlas. These styles should adhere to the following interface:
+
+```ts
+interface SafeAreaViewStyle {
+    unsafeAreaTop?: {
+        backgroundColor?: string;
+    };
+    unsafeAreaBottom?: {
+        backgroundColor?: string;
+    };
+    container?: ViewStyle;
+}
+```
+
+An example of a default custom style:
+
+```js
+export const com_mendix_widget_native_safeareaview_SafeAreaView = {
+    unsafeAreaTop: {
+        backgroundColor: "#FFF"
+    },
+    unsafeAreaBottom: {
+        backgroundColor: "#FFF"
+    },
+    container: {
+        flex: 1,
+        backgroundColor: "#FFF"
+    }
+};
+```
+
+An example of a class custom style:
```

Comment:
"custom style class" maybe?

Reply:
Done, also changed it for the default custom style class